core/menuitem.ts

```diff
@@ -185,15 +183,6 @@ export class MenuItem {
    */
   setHighlighted(highlight: boolean) {
```

Comment:
Can you actually delete this method entirely? It's internal and `this.highlight` is private, so removing it isn't a breaking change =)